### DISCUSSION TITLE
How can typeorm be used with nuxt3?

### DISCUSSION THREAD

**Discussion by bbhxwl** [nuxt/nuxt#23684] [Questions]
### Environment

- Operating System: Windows_NT                                                                                                                                         
- Node Version:     v18.17.1                                                                                                                                           
- Nuxt Version:     3.7.4                                                                                                                                              
- CLI Version:      3.9.0                                                                                                                                              
- Nitro Version:    2.6.3
- Package Manager:  yarn@1.22.19
- Builder:          -
- User Config:      -
- Runtime Modules:  -
- Build Modules:    -

### Reproduction

![image](https://github.com/nuxt/nuxt/assets/26525599/3274758a-c499-4b32-b830-61412799cc11)

### Describe the bug

Unable to use according to the tutorial definition, error. Can't nuxt3 use typeorm?

**Comment by capopovici** [comment]
Add this to your /server/tsconfig.json
```
{
  "extends": "../.nuxt/tsconfig.server.json",
  "compilerOptions": {
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
  }
}
```
